cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.ql

```diff
@@ -0,0 +1,66 @@
+/**
+ * @name Comma before misleading indentation
+ * @description The expressions before and after the comma operator can be misread because of an unusual difference in start columns.
+ * @kind problem
+ * @id cpp/comma-before-misleading-indentation
+ * @problem.severity warning
+ * @precision medium
+ * @tags maintainability
+ *       readability
+ */
+
+import cpp
+import semmle.code.cpp.commons.Exclusions
+
+/**
+ * Holds if this is an implicit `this`.
+ *
+ * ThisExpr.isCompilerGenerated() is currently not being extracted, so use a heuristic.
```

Comment:
We should create an internal issue for this. This is most likely a simple oversight.